modules/self-test/self-test.lua

```diff
@@ -41,12 +41,35 @@
 		m.loadTestsFromManifests()
 		m.detectDuplicateTests = false
 
-		local test, err = m.getTestWithIdentifier(_OPTIONS["test-only"])
-		if err then
-			error(err, 0)
+		local tests = {}
+		local isAction = true
+		for i, arg in ipairs(_ARGV) do
+			-- Skip any _OPTIONS
+			if not arg:startswith("/") and not arg:startswith("--") then
```

Comment:
Could this use [_ARGS](https://github.com/premake/premake-core/wiki/_ARGS) instead?

Reply:
I don't see why not, I wasn't aware of `_ARGS` so that's cool. The docs aren't clear about what it contains, does it exclude `_OPTIONS`?

Reply:
It does exclude the options, I'll update the PR to use `_ARGS` give me a minute 😄

Reply:
The PR has been updated to use `_ARGS`, much better! Thanks for the tip @starkos!